plugin/main.cpp

```diff
@@ -19,7 +19,8 @@
 Q_DECL_EXPORT int main(int argc, char *argv[])
 {
     QLoggingCategory::setFilterRules(QStringLiteral("geoclue.provider.mlsdb.debug=false\n"
-                                                    "geoclue.provider.mlsdb.position.debug=false"));
+                                                    "geoclue.provider.mlsdb.position.debug=false\n"
+                                                    "geoclue.provider.mlsdb.online.debug=false"));
```

Comment:
mlsdbonlinelocator.cpp: Q_LOGGING_CATEGORY(lcGeoclueMlsdbOnline, "geoclue.provider.mlsdb.online")

Two parameter docs: "By default, all message types are enabled.". Three parameter version would allow to indicate default filtering level.

LGTM, but including message levels on category declarations could be cleaner?